src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import './navbar.css';
+import ScrollTop from '../ScrollTop.jsx';
+
+const Navbar = () => {
+    const [scrolled, setScrolled] = useState(false);
+    const [burgerOpen, setBurgerOpen] = useState(false);
+    const location = useLocation();
+
+    useEffect(() => {
+        const handleScroll = () => {
+            const offset = window.scrollY;
+            if(offset > 50){
+                setScrolled(true);
+            }
+            else{
+                setScrolled(false);
+            }
+        };
+        window.addEventListener('scroll', handleScroll);
+
+        return () => {
+            window.removeEventListener('scroll', handleScroll);
+        }
+    }, []);
+
+    let navbar = 'navbar';
+    if(scrolled){
+        navbar += ' navbar__scrolled';
+    }
+
+    const burgerClick = () => {
+        setBurgerOpen(!burgerOpen);
+    }
+
+    const isActive = (path) => location.pathname === path ? 'active' : '';
+
+    return (
+        <div className={navbar}>
+            <ScrollTop></ScrollTop>
```

Comment:
Is this component visible or used?

Reply:
It's used so that when we changed to another page of the website, the page will start again from the top. If we don't use this component, we will end up opening at the last section we scrolled before at that page.